### ISSUE TITLE
Bad link creation line 25 ( src -> /home/portia/portia/src )

### ISSUE THREAD

**Issue by TMLutas** [tytek2012/Portia_Installer_Script#5]
Line 25 creates a link that goes nowhere

sudo ln -s ~/portia/src ~/portia/slyd/

This creates the following link to a directory that doesn't exist

src -> /home/portia/portia/src 

Is this desired behavior and I'm doing something wrong elsewhere? Or is this an error in the script?

**Comment by tytek2012** [comment]
did you make your user account with the name portia ?

**Comment by tytek2012** [comment]
maybe i could use a /home/$userName/portia/slyd/

**Comment by TMLutas** [comment]
Yes, I made a user account with the name portia. I'm just not sure what the link is supposed to be doing. I'm going to be posting up the step by step that worked for me in another issue.
